privaterelay/tests/mgmt_update_phone_remaining_stats_tests.py

```diff
@@ -266,3 +267,40 @@ def test_phone_subscriber_with_subscription_end_date_sooner_than_31_days_since_r
     assert relay_number.remaining_texts == settings.MAX_TEXTS_PER_BILLING_CYCLE
     assert relay_number.remaining_minutes == settings.MAX_MINUTES_PER_BILLING_CYCLE
     assert relay_number.remaining_seconds == settings.MAX_MINUTES_PER_BILLING_CYCLE * 60
+
+
+def test_phone_subscriber_with_subscription_end_date_after_reset_phone_limits_updated(
+    phone_user,
+):
+    datetime_now = datetime.now(timezone.utc)
+    datetime_first_of_march = datetime_now.replace(month=3, day=4)
+    profile = Profile.objects.get(user=phone_user)
+    profile.date_subscribed_phone = datetime_now.replace(month=1, day=2)
+    new_subscription_start_and_previous_reset_date = datetime_now.replace(
+        month=2, day=1
```

Comment:
(suggestion, blocking) This might be a problem if the test is being ran on February. Consider creating the dates as a relative.

Reply:
Not entirely sure if this would be an issue, since the other tests are also using hard coded dates similar to the one above. To run it for February someone would need to change the date in that function I believe. Let me know if I'm missing something here.

Reply:
I see, you were following what the other tests were doing. That was me and yes you are right that this is not an issue since the `datetime` is also mocked to be a static date that is returned.